packages/ddp-client/src/livechat/LivechatClientImpl.ts

```diff
@@ -324,6 +324,13 @@ export class LivechatClientImpl extends DDPSDK implements LivechatStream, Livech
 		});
 	}
 
+	async sendUiInteraction(
+		payload: OperationParams<'POST', '/apps/ui.interaction/:id'>,
+		{ appId, visitorToken }: { appId: string; visitorToken: string },
```

Comment:
Instead of sending the token via params, use `this.token`, you can even check to see if the token exists and send the appropriate error message

Reply:
This was helpful! Thanks :hugs: